lib/Auth.js

```diff
@@ -87,6 +87,8 @@ function useAuthProvider() {
         window.localStorage.removeItem('mx_hs_url');
         window.localStorage.removeItem('mx_user_id');
         window.localStorage.removeItem('mx_device_id');
+        window.localStorage.clear()
```

Comment:
This does not make sense. Why manually remove certain items in the lines above, when you clear the whole storage afterwards?

I also prefer to manually remove only those entries, which I also purposefully set beforehand. If there's something in the local storage that shouldn't be there, then let's try to find a place in the code that makes more sense to remove it from there.

Reply:
I agree, either just `clear()` or individual items. Personally I prefer just `clear()` since it's only one line of code but it's not a strong preference.

I will fix this in a future commit and go back to individual items.